### PR TITLE
Fetch the browser logos via `npm`

### PR DESCRIPTION
**Pull request by alrra**
Fetch the browser logos via `npm` (See also / Fix: #1414)

### CODE REVIEW

File: webapp/package.json [hunk 1]
```diff
@@ -7,8 +7,24 @@
...
+    "@browser-logos/firefox": "^3.0.6",
+    "@browser-logos/firefox-beta": "^4.1.1",
+    "@browser-logos/firefox-developer-edition": "^3.0.3",
+    "@browser-logos/firefox-nightly": "^3.0.3",
+    "@browser-logos/safari": "^1.0.6",
+    "@browser-logos/safari-technology-preview": "^1.0.7",
+    "@browser-logos/uc": "^1.0.5",
     "@polymer/test-fixture": "4.0.2",
     "babel-eslint": "10.0.3",
+    "cpx": "^1.5.0",
```
Comment (Hexcles) [inline webapp/package.json]: Question: does the **cpx** package provide the **cpy** command? I'd expect `cpx` to provide `cpx`, while `cpy-cli` provides `cpy`, no?

Comment (alrra) [reply webapp/package.json]: @Hexcles Nice catch! Should be fixed (I was trying a few things, and it seems I forgot to update that).

**Comment by alrra** [comment]
FYI @foolip ⬆

File: webapp/package.json [hunk 2]
```diff
@@ -7,8 +7,24 @@
...
   },
   "devDependencies": {
+    "@browser-logos/chrome": "^1.0.15",
+    "@browser-logos/chrome-beta": "^2.1.4",
+    "@browser-logos/chrome-canary": "^1.1.7",
+    "@browser-logos/chrome-dev": "^2.1.7",
+    "@browser-logos/edge": "^1.0.10",
+    "@browser-logos/edge-beta": "^1.0.11",
+    "@browser-logos/edge-canary": "^1.0.9",
+    "@browser-logos/edge-dev": "^1.0.9",
```
Comment (alrra) [inline webapp/package.json]: If you want the new `Edge` logos, update these to `v2`.

Comment (alrra) [reply webapp/package.json]: @foolip Done!

**Pull request merged**